### DISCUSSION TITLE
RoderickShao_notebook/%E8%AE%A1%E7%AE%97%E6%9C%BA%E5%9F%BA%E7%A1%80%E8%AF%BE/%E9%AB%98%E7%BA%A7%E6%95%B0%E6%8D%AE%E7%BB%93%E6%9E%84%E4%B8%8E%E7%AE%97%E6%B3%95%E5%88%86%E6%9E%90/Lecture%2014/

### DISCUSSION THREAD

**Discussion by giscus** [RoderickShao/RoderickShao_notebook#10] [Announcements]
# RoderickShao_notebook/%E8%AE%A1%E7%AE%97%E6%9C%BA%E5%9F%BA%E7%A1%80%E8%AF%BE/%E9%AB%98%E7%BA%A7%E6%95%B0%E6%8D%AE%E7%BB%93%E6%9E%84%E4%B8%8E%E7%AE%97%E6%B3%95%E5%88%86%E6%9E%90/Lecture%2014/

RoderickShao's NoteBook

https://roderickshao.github.io/RoderickShao_notebook/%E8%AE%A1%E7%AE%97%E6%9C%BA%E5%9F%BA%E7%A1%80%E8%AF%BE/%E9%AB%98%E7%BA%A7%E6%95%B0%E6%8D%AE%E7%BB%93%E6%9E%84%E4%B8%8E%E7%AE%97%E6%B3%95%E5%88%86%E6%9E%90/Lecture%2014/

**Comment by Provence-24** [comment]
下次笔记不要抄别人了 不然成习惯了。我来跨界监督你一下【坏笑】

**Comment by Provence-24** [comment]
Maximum-Finding中要十分留意 每一次划分后，子问题是怎么解决的以及子问题是怎么合并的。这些都需要非常注意。逻辑很深，一环扣一环。最后一种方法反而比较独立，只跟法2有关系。
